config/examples/Creality/Ender-3/BigTreeTech SKR Mini E3 2.0/Configuration.h

```diff
@@ -1611,7 +1611,7 @@
   /**
    * Enable the G26 Mesh Validation Pattern tool.
    */
-  //#define G26_MESH_VALIDATION
+  #define G26_MESH_VALIDATION
```

Comment:
This seems like an unrelated change.

Reply:
yep, generated pattern is borderline useless...

Reply:
turned back off in the latest code.